options/rtdl/generic/linker.cpp

```diff
@@ -683,7 +683,7 @@ void ObjectRepository::_parseDynamic(SharedObject *object) {
 		case DT_TLSDESC_PLT: case DT_TLSDESC_GOT:
 			break;
 		default:
-			mlibc::panicLogger() << "Unexpected dynamic entry "
+			mlibc::infoLogger() << "Unexpected dynamic entry "
```

Comment:
I'd rather not change this, we should just add the missing dynamic entry into the code and add logging to say we don't handle it properly.

Reply:
They seem to be values in the os-specific area that are specific to wine .so dlls, so maybe it should just log the unknown os specific entries but otherwise ignore them?